specifications/api/swagger.yaml

```
@@ -4781,3 +5025,72 @@ paths:
           required: false
           description: check this url for health instead of the currently configured one
       responses: *responsesGetSmashHealth
+
+  /shared-wallets:
+    post:
+      operationId: postSharedWallet
+      tags: ["Shared Wallets"]
+      summary: Create
+      description: |
+        <p align="right">status: <strong>unstable</strong></p>
+
+        Create a shared wallet from an account public key, its index and script templates.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: *ApiSharedWalletPostData
+      responses: *responsesPostSharedWallet
+
+  /shared-wallets/{walletId}:
+    get:
+      operationId: getSharedWallet
+      tags: ["Shared Wallets"]
+      summary: Get
+      description: |
+        <p align="right">status: <strong>unstable</strong></p>
+      parameters:
+        - *parametersWalletId
+      responses: *responsesGetSharedWallet
+
+    delete:
+      operationId: deleteSharedWallet
+      tags: ["Shared Wallets"]
+      summary: Delete
+      description: |
+        <p align="right">status: <strong>unstable</strong></p>
+      parameters:
+        - *parametersWalletId
+      responses: *responsesDeleteWallet
+
+  /shared-wallets/{walletId}/payment-script-template:
+    patch:
+      operationId: patchSharedWalletInPayment
+      tags: ["Shared Wallets"]
+      summary: Update
+      description: |
+        <p align="right">status: <strong>unstable</strong></p>
+      parameters:
+        - *parametersWalletId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: *ApiSharedWalletPatchData
+      responses: *responsesPatchSharedWallet
+
+  /shared-wallets/{walletId}/delegation-script-template:
+    patch:
+      operationId: patchSharedWalletInDelegation
+      tags: ["Shared Wallets"]
+      summary: Update
```

```suggestion
      summary: Update Delegation
```
(for the same reason :point_up_2: )

---

done